### PR TITLE
FIX: stretch is now in archive deb repo of postgres

### PR DESCRIPTION
**Pull request by damdam-s**

### CODE REVIEW

**Review by yvaucher — APPROVED** [review]

**Review by cyrilmanuel — APPROVED** [review]

**Review by leemannd — APPROVED** [review]

**Review by leemannd — COMMENTED** [review]
You may want to update history

File: install/postgres.sh [hunk 1]
```diff
@@ -16,6 +16,10 @@ if [ $OS_CODENAME = "jessie" ]
 then
     APT_REPO="apt-archive.postgresql.org"
 fi
+if [ $OS_CODENAME = "stretch" ]
```
Comment (p-tombez) [inline install/postgres.sh:19]: Why not use an `OR` with the test above ?

Comment (damdam-s) [reply install/postgres.sh:19]: not so used to bash scripts and need to have a quick solution

Comment (p-tombez) [reply install/postgres.sh:19]: ```if [ $OS_CODENAME = "jessie" ] || [ $OS_CODENAME = "stretch" ]```

Comment (damdam-s) [reply install/postgres.sh]: done in last commit

**Review by p-tombez — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Comment by vrenaville** [comment]
@damdam-s I propose this PR https://github.com/camptocamp/docker-odoo-project/pull/223

**Pull request merged**